src/store.rs

```diff
@@ -232,6 +238,7 @@ type Key = String;
 pub struct Value {
     pub data: Bytes,
     pub expires_at: Option<Instant>,
+    pub inserted_at: Instant,
```

Comment:
created_at? I've always seen/used created_at and updated_at on db records.

Reply:
Yeah, that's better